.github/ISSUE_TEMPLATE/---releng-checklist.md

```diff
@@ -7,6 +7,9 @@ title: 'Releng checklist - vX.X'
 
 ##### Sprint Release Owner: _[name here]_
 
+#### Monday #1
+- [ ] Update this issue with all the relevant fields
```

Comment:
What is this referring to?

Reply:
I can clarify - it's referring to all the `vX.X` fields throughout the Releng checklist.

Reply:
Do you think it's necessary to have a checklist item for that? I think as long as the title is updated with the correct version, those fields should be clear.